donate/templates/forms/_field.html

```diff
@@ -1,4 +1,4 @@
-<div class="form-item__wrapper">
+<div class="form-item__wrapper {% if widget_type == 'select' or widget_type == 'lazy_select' %}form-custom-select{% endif %}">
```

Comment:
I'm wrong - we need this because `CountryField` uses a custom widget. See https://github.com/SmileyChris/django-countries/blob/master/django_countries/widgets.py#L52

Reply:
turns out we do need to have the lazy_select widget type, so as agreed with Samir, we're going to keep this